src/proof.js

```diff
@@ -155,3 +156,80 @@ function areNodesEqual(node1, node2) {
   }
   return true
 }
+
+/**
+ * Extension on of the IPLD View that adds accessors for the data model fields.
+ *
+ * @implements {API.ProofDataView}
+ * @extends {View<API.ProofData, API.ProofDataLayout, typeof CBOR.code, typeof SHA256.code>}
+ */
+class ProofDataView extends View {
+  get path() {
+    return this.model.path
+  }
+  get at() {
+    return this.model.at
+  }
+}
+
+/**
+ * Takes data model and returns a data layout used in the CBOR encoding.
+ *
+ * @param {API.ProofData} source
+ * @returns {API.ProofDataLayout}
+ */
+export const into = ({ path, at }) => [BigInt(at), path]
+
+/**
+ * Takes data layout used in CBOR encoding and returns a corresponding data
+ * model.
+ *
+ * @param {API.ProofDataLayout} layout
+ * @returns {API.ProofData}
+ */
+export const from = ([at, path]) => ({ path, at: BigInt(at) })
+
+/**
+ * Takes data layout expected by the CBOR encoding and returns CBOR encoded
+ * bytes corresponding to it.
+ *
+ * @param {API.ProofDataLayout} proof
+ * @returns {API.ByteView<API.ProofDataLayout, CBOR.code>}
+ */
+export const encode = (proof) => CBOR.encode(proof)
+
+/**
+ * Takes CBOR encoded bytes for the data layout and returns decoded data.
+ *
+ * @param {API.ByteView<API.ProofDataLayout, CBOR.code>} bytes
+ * @returns {API.ProofDataLayout}
+ */
+export const decode = (bytes) => {
+  const [index, path] = CBOR.decode(bytes)
+  // Note we do bigint conversion because DAG CBOR will use JS Number if
+  // Int is small enough
+  return [BigInt(index), path]
```

Comment:
`at` for consistency with naming in the `into` an d `from` implementations above
```suggestion
  const [at, path] = CBOR.decode(bytes)
  // Note we do bigint conversion because DAG CBOR will use JS Number if
  // Int is small enough
  return [BigInt(at), path]
```

Reply:
Good catch, I just renamed `index` to `at` so we won't have `proof.index.index` and probably missed this one